README.md

```diff
@@ -419,7 +419,8 @@ const authentication = {
   // "test" could also be a function
   test: {
     url: 'https://example.com/api/accounts/me.json'
-  }
+  },
+  connectionLabel: '{{bundle.authData.username}}' // Can also be a function, check digest auth for an example
```

Comment:
It's small, but what do you think about saying "check digest auth below" so folks know we mean to stay in this section? When I read this out of context, there were a number of ideas I had for what "check digest auth" could mean.